mempool/mempool.go

```diff
@@ -65,28 +67,35 @@ func NewMempoolMonitor(collector *mempoolDataCollector,
 	}
 }
 
-type TicketsDetails []*apitypes.TicketDetails
+// TicketsDetails localizes apitypes.TicketsDetails
+type TicketsDetails apitypes.TicketsDetails
 
+// Len returns the lenght of TicketsDetails
 func (tix TicketsDetails) Len() int {
 	return len(tix)
 }
 
+// Swap swaps TicketsDetails elements at i and j
 func (tix TicketsDetails) Swap(i, j int) {
 	tix[i], tix[j] = tix[j], tix[i]
 }
 
+// ByFeeRate models TicketsDetails sorted by fee rates
 type ByFeeRate struct {
 	TicketsDetails
 }
 
+// Less returns TicketsDetails[i].FeeRate < TicketsDetails[j].FeeRate
```

Comment:
more semantically, something like `Less compares fee rates by rate_i < rate_j`.
I'd prefer not to mirror the code.